drivers/network/tcpip/tcpip/fileobjs.c

```diff
@@ -509,16 +512,16 @@ NTSTATUS FileOpenAddress(
 
   case IPPROTO_UDP:
       TI_DbgPrint(MID_TRACE,("Allocating udp port\n"));
-      AddrFile->Port =
-	  UDPAllocatePort(Address->Address[0].Address[0].sin_port);
+      nPort = UDPAllocatePort(Address->Address[0].Address[0].sin_port);
 
       if ((Address->Address[0].Address[0].sin_port &&
            AddrFile->Port != Address->Address[0].Address[0].sin_port) ||
```

Comment:
This should be a comparison with `nPort` now ?

Reply:
Yes, my mistake.